site/en/blog/_example/index.md

```diff
@@ -51,6 +51,11 @@ hero: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
 # Required if there is a hero or social image
 alt: >
   An alternative text description of your hero or social image.
```

Comment:
Please add information about social images to the handbook too, next to hero images https://developer.chrome.com/docs/handbook/how-to/add-a-blog-post/#hero-images

Reply:
I added the information about social images here: https://github.com/GoogleChrome/developer.chrome.com/pull/4981/files#diff-884a22587fc7726d735ce53ba4ac18e41f40ee70593213c811e573dfa3eaf428